examples/platform/asr/ASRUtils.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <app-common/zap-generated/attributes/Accessors.h>
+#include <app-common/zap-generated/ids/Attributes.h>
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app/server/Dnssd.h>
+#include <app/server/OnboardingCodesUtil.h>
+#include <app/server/Server.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
why does this file exist? It has empty content except including a bunch of other includes

also the name is `utils` which is not useful (like helpers/support and such).

Reply:
rename ASRUtils to matter_shell to  provides the Shell implementation of Matter, it can be also used in ASR AT Command implementation